coverage/.resultset.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
I found that after you `git add coverage/`, if you run `git stash` it will "stash" it away and you don't have to commit it